File: create_favicon.py
from PIL import Image
import os

# Charger le logo
logo = Image.open('logostylrougeok.png').convert('RGBA')

# Créer fond blanc carré 512x512
favicon = Image.new('RGBA', (512, 512), (255, 255, 255, 255))

# Redimensionner le logo à 440x440 centré
logo = logo.resize((440, 440), Image.LANCZOS)

# Coller au centre
favicon.paste(logo, (36, 36), logo)

# Sauvegarder
favicon.save('favicon.png')
print("favicon.png créé")

File: create_favicon.mjs
import Jimp from 'jimp'

const logo = await Jimp.read('logostylrougeok.png')
logo.resize({ w: 440, h: 440 })

const favicon = new Jimp({ width: 512, height: 512, color: 0xffffffff })
favicon.composite(logo, 36, 36)

await favicon.write('favicon.png')
console.log('favicon.png créé (512x512 fond blanc)')
